src/gui/qgsnewgeopackagelayerdialog.cpp

```diff
@@ -64,35 +64,36 @@ QgsNewGeoPackageLayerDialog::QgsNewGeoPackageLayerDialog( QWidget *parent, Qt::W
   mAddAttributeButton->setIcon( QgsApplication::getThemeIcon( QStringLiteral( "/mActionNewAttribute.svg" ) ) );
   mRemoveAttributeButton->setIcon( QgsApplication::getThemeIcon( QStringLiteral( "/mActionDeleteAttribute.svg" ) ) );
 
-  mGeometryTypeBox->addItem( tr( "Non spatial" ), wkbNone );
-  mGeometryTypeBox->addItem( tr( "Point" ), wkbPoint );
-  mGeometryTypeBox->addItem( tr( "Line" ), wkbLineString );
-  mGeometryTypeBox->addItem( tr( "Polygon" ), wkbPolygon );
-  mGeometryTypeBox->addItem( tr( "Multi point" ), wkbMultiPoint );
-  mGeometryTypeBox->addItem( tr( "Multi line" ), wkbMultiLineString );
-  mGeometryTypeBox->addItem( tr( "Multi polygon" ), wkbMultiPolygon );
+  QString path = QgsApplication::defaultThemePath();
+  mGeometryTypeBox->addItem( QIcon( path + "/mIconTableLayer.svg" ), tr( "Non spatial" ), wkbNone );
```

Comment:
Should we rename this to "No Geometry" to follow the "save as vector layer" dialog?

Reply:
Yeah that makes more sense than "non spatial geometry" :)